lib/saml_idp/xml_security.rb

```diff
@@ -149,7 +153,7 @@ def validate_doc_params_signature(base64_cert, soft = true, params)
         )
       end
 
-      def validate_doc_embedded_signature(base64_cert, soft = true)
+      def validate_doc_embedded_signature(base64_cert, soft = true, no_namespace_enabled: false)
```

Comment:
so the plan is to call into `SamlIdp::XmlSecurity` directly from the IDP? Or do we need to expose a second method on `SamlIdp::Request`?

Reply:
the caller method is `request.matching_cert`! it's not currently in use with the IdP, i made it to allow us to remove `service_provider.matching_cert`, but will come in handy here before we make that change. i think this logging will also allow us to be confident in eventually swapping out `request.matching_cert`

Reply:
ok perf! thx for explaining